packages/vue/src/components/password-control.vue

```diff
@@ -54,34 +56,62 @@ export default {
 
 <template>
   <base-wrapper
-    class="amplify-flex amplify-field amplify-textfield amplify-passwordfield amplify-authenticator__column"
+    :class="
+      classNames(
+        ComponentClassName.Flex,
+        ComponentClassName.Field,
+        ComponentClassName.TextField,
+        ComponentClassName.PasswordField
+      )
+    "
```

Comment:
```suggestion
    :class="
      [
        ComponentClassName.Flex,
        ComponentClassName.Field,
        ComponentClassName.TextField,
        ComponentClassName.PasswordField
      ]
    "
```

You can list class names in Vue as an [array](https://vuejs.org/guide/essentials/class-and-style.html#binding-html-classes).

Reply:
Do we need the `classnames` library at all for vue then?

Reply:
TIL, thanks @ErikCH! Probably not, I'll go ahead and remove it xD